imagersite/imager_images/tests.py

```diff
@@ -129,6 +152,32 @@ def test_user_can_see_its_photo_description(self):
         one_album = Album.objects.first()
         self.assertEqual(one_album.photos.first().description, one_album.photos.first().description)
 
+    def test_user_can_get_to_photo_creation(self):
+        """Test user can see photo form."""
+        user = User.objects.first()
+        self.client.force_login(user)
+        response = self.client.get('/images/photos/add/')
+        self.assertEqual(response.status_code, 200)
+
+    def test_user_cant_get_to_photo_creation_if_not_logged_in(self):
+        """Test user can see photo form."""
+        response = self.client.get('/images/photos/add/')
+        self.assertEqual(response.status_code, 302)
+
+    @override_settings(MEDIA_ROOT=os.path.join(settings.BASE_DIR, 'test_stuff'))
+    def test_create_photo(self):
+        """Test for photo creation."""
+        os.system(f'mkdir {os.path.join(settings.BASE_DIR, "test_stuff")}')
+        user = User.objects.first()
+        self.client.force_login(user)
+        with open(os.path.join(settings.BASE_DIR, 'imagersite/static/camera.png'), 'rb') as f:
+            image = ContentFile(f.read())
+        image.name = 'test.png'
+
+        response = self.client.post('/images/photos/add/', {'albums': [Album.objects.first().id], 'title': 'stuff', 'description': 'stuff', 'image': image, 'published': 'PRIVATE'})
+        self.assertEqual(response.status_code, 302)
+        os.system(f'rm -rf {os.path.join(settings.BASE_DIR, "test_stuff")}')
```

Comment:
You can use `shutil.rmtree()` for this command (`os.rmdir()` only works if the directory is empty).